src/js/control-bar/progress-control/time-tooltip.js

```diff
@@ -52,58 +52,39 @@ class TimeTooltip extends Component {
    *        from the left edge of the {@link SeekBar}
    */
   update(seekBarRect, seekBarPoint, content) {
-    const tooltipRect = Dom.findPosition(this.el_);
-    const playerRect = Dom.getBoundingClientRect(this.player_.el());
-    const seekBarPointPx = seekBarRect.width * seekBarPoint;
+    this.write(content);
```

Comment:
if we do a `return` after `this.write(content); ` we have same behavior as of today.